projectile.el

```diff
@@ -251,7 +251,16 @@ is set to 'alien'."
           (const :tag "Recently opened files" recentf)
           (const :tag "Recently active buffers, then recently opened files" recently-active)
           (const :tag "Access time (atime)" access-time)
-          (const :tag "Modification time (mtime)" modification-time)))
+          (const :tag "Modification time (mtime)" modification-time)
+          (const :tag "Path length" path-length)
+          (const :tag "Custom method" custom-method)))
+
+(defcustom projectile-sort-custom-method nil
+  "The sort method to use for custom sorting.
+
+This should be the name of a predicate method usable by `cl-sort'"
+  :group 'projectile
+  :type 'symbol)
```

Comment:
You should also add a package-version property here saying 2.1

Reply:
i did 2.1.0, since the others were all 3 digit